src/book/garden-circle/contributing.md

```diff
@@ -137,6 +137,27 @@ These workflows include one or more [jobs](https://docs.github.com/en/actions/us
 We define GitHub Actions work under the [`.github`](https://github.com/software-gardening/almanac/tree/main/.github) directory.
 We suggest the use of [`act`](https://github.com/nektos/act) to help test GitHub Actions work during development.
 
+### Releases
+
+We utilize [semantic versioning](https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning) ("semver") in order to distinguish between major, minor, and patch releases.
+We publish source code by using [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) available [here](https://github.com/software-gardening/almanac/releases).
+We publish a related Python package through the [Python Packaging Index (PyPI)](https://pypi.org/) available [here](https://pypi.org/project/almanack/).
+
+#### Release Publishing Process
+
+Several manual and automated steps are involved with publishing Software Gardening Almanack releases.
+See below for an overview of how this works.
+
+Notes about [semantic version](https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning) (semver) specifications:
```

Comment:
This line feels a bit out of place given the previous "see below..." language

You immediately point to a different resource for someone to view? perhaps an easy fix would be to add this sentence later, after describing specs

Reply:
Thanks, great point - I'll move things around a bit to help with this.

Reply:
Reorganized this with a further commit.